_data/navigation.yaml

```diff
@@ -14,8 +14,12 @@
       new_window: false
     - name: Service Mesh Interface Conformance
       link: /service-mesh-interface
-      new_window: true    
-      external: true        
+      new_window: false  
```

Comment:
Why the following change?

Reply:
@kushthedude The new project page is internal to the layer5.io site, so we will have this link open in the _parent, not _blank target.